pootle/apps/pootle_misc/checks.py

```diff
@@ -1008,13 +1010,17 @@ def get_qualitycheck_schema(path_obj=None):
         Category.COSMETIC: _("Cosmetic"),
         Category.FUNCTIONAL: _("Functional"),
         Category.EXTRACTION: _("Extraction"),
-        Category.NO_CATEGORY: _("Other"),
+        Category.NO_CATEGORY: _("Other")
     }
 
+    cat_id_keys = category_ids.keys()
+    cat_id_values = category_ids.values()
+
     for check, cat in checks.items():
         if cat not in d:
             d[cat] = {
                 'code': cat,
+                'name': cat_id_keys[cat_id_values.index(cat)],
```

Comment:
yep agreed - id been thinking about this

i also figured it would be good to have a get_category_name_from_code() func or similar